tests/Integration/Adapter/Artax/NetscapeCookieFileJarTest.php

```diff
@@ -40,27 +42,33 @@ public function testLoadCookies()
 
         $jar = new NetscapeCookieFileJar($path);
 
-        $this->assertCount(4, $jar->getAll()['.php-fig.org']['/']);
+        $this->assertCount(4, \array_filter($jar->getAll(), static function (ResponseCookie $cookie) {
+            return $cookie->getDomain() !== '.php-fig.org' || $cookie->getPath() !== '/';
+        }));
 
         $this->assertEquals(
             '-5061451:',
-            $jar->get('.google.com', '', 'OGP')[0]->getValue()
+            (yield $jar->get((new Request('https://google.com/'))->getUri()))[0]->getValue()
```

Comment:
Didn't realise you could use `yield` like this

Reply:
I guess I should refactor this, it's rather hard to read.